SingularityBase/src/main/java/com/hubspot/singularity/SingularityAuthorizationScope.java

```diff
@@ -4,5 +4,6 @@
   READ,
   WRITE,
   ADMIN,
-  DEPLOY
+  DEPLOY,
+  EXEC // run, bounce, kill, pause, scale
```

Comment:
Since we are adding granularity, would it be too much harder to actually make each of these their own separate scope? Would let us much more easily divide things up later if needed